packages/devtools_shared/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: devtools_shared
 description: Package of shared structures between devtools_app and devtools_server.
 
-version: 0.2.2
+version: 0.1.16
```

Comment:
arg. we need to move all the packages to 0.2.3 to keep it consistent.

Reply:
Done.